roles/container-engine/docker/tasks/main.yml

```diff
@@ -109,6 +109,19 @@
     dest: "{{ yum_repo_dir }}/docker.repo"
   when: ansible_distribution in ["CentOS","RedHat"] and not is_atomic
 
+# FIXME: maybe better check for enabled product
```

Comment:
Maybe using `zypper -xqt pd -i` with xpath filter (only if it does not require extra modules) could be an option...